### PR TITLE
Changed colors to use 18F colors

### PR DESCRIPTION
**Pull request by KimberlyMunoz**
Changing more colors across CF repos.
## Review
- @Scotchester 
- @jimmynotjim 
## Preview

image

image

### CODE REVIEW

File: src/cf-expandables.less [hunk 1]
```diff
@@ -37,27 +37,27 @@
...
 // .expandable__padded
-@expandable__padded-bg:         lighten(#905c8a, 45%);
+@expandable__padded-bg:         #e1f3f8; // $color-primary-alt-lightest
 @expandable__padded-bg-hover:   lighten(@expandable__padded-bg, 3%);
 @expandable__padded-divider:    darken(@expandable__padded-bg, 5%);
 
 // .expandable-group
-@expandable-group_header-text:  darken(#895983, 15%);
-@expandable-group_header-bg:    lighten(#905c8a, 45%);
+@expandable-group_header-text:  #000;
```
Comment (jimmynotjim) [inline src/cf-expandables.less]: Should this be `@text` since it imports cf-core anyway?

**Comment by jimmynotjim** [comment]
One small note, but otherwise :+1:

File: src/cf-expandables.less (hunk 1 above)
Comment (KimberlyMunoz) [inline src/cf-expandables.less]: Changed in e39f02f

**Comment by KimberlyMunoz** [comment]
Thanks for the review! Changes made.

File: src/cf-expandables.less [hunk 2]
```diff
@@ -37,27 +37,27 @@
   - less
 */
 
-// Color variables 
+// Color variables
```
Comment (Scotchester) [inline src/cf-expandables.less:40]: Include the comment on the origin of the `$color-` variables?

**Comment by KimberlyMunoz** [comment]
Ack. Actually, I need to change the breakpoint variables first. One sec.

File: src/cf-expandables.less (hunk 2 above)
Comment (KimberlyMunoz) [inline src/cf-expandables.less:40]: Fixed in e472490

**Comment by Scotchester** [comment]
:+1: here!

**Comment by Scotchester** [comment]
Merge now or bag it & tag it for release? I don't have a preference, it's up to you.

**Comment by Scotchester** [comment]
:bell: @KimberlyMunoz

**Comment by KimberlyMunoz** [comment]
Merge when ready and I can tag tomorrow morning

**Pull request merged**